ocamlbuild/plan.sh

```diff
@@ -0,0 +1,16 @@
+pkg_name=ocamlbuild
+pkg_origin=core
+pkg_version="0.11.0"
+pkg_description="OCamlbuild is a generic build tool, that has built-in rules for building OCaml library and programs."
+pkg_upstream_url="https://github.com/ocaml/ocamlbuild"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('LGPL')
+pkg_source="https://github.com/ocaml/ocamlbuild/archive/${pkg_version}.tar.gz"
+pkg_shasum="1717edc841c9b98072e410f1b0bc8b84444b4b35ed3b4949ce2bec17c60103ee"
+pkg_deps=(core/glibc core/ocaml core/make core/gcc)
```

Comment:
This fails to run:

```
[48][default:/src:0]# hab pkg exec core/ocamlbuild ocamlbuild
+ mkdir /src/_build
/bin/sh: mkdir: command not found
Command exited with code 127.
Failure:
  Error during command "mkdir /src/_build": Ocamlbuild_pack.My_std.Exit_with_code(10).
```

If you add `core/coreutils` to get `mkdir`, it's missing `tput`. 

I got it to work by changing things to look like this:

```
pkg_deps=(
  core/glibc
  core/ocaml
  core/coreutils
  core/ncurses
)
pkg_build_deps=(
  core/gcc
  core/make
)
```